### ISSUE TITLE
has_distribution does not work for multiple distributions?

### ISSUE THREAD

**Issue by peterdesmet** [trias-project/trias#9]
Function works for taxa with one distribution: https://api.gbif.org/v1/species/139334263/distributions

```r
has_distribution(
  139334263,
  country = "CA"
)
```

But not for multiple distributions: https://api.gbif.org/v1/species/139334288/distributions

```r
has_distribution(
  139334288, 
  country = "CA"
)
```

Error:

```
Error in intersect_data_frame(x, y) : 
  not compatible: Cols in y but not x: `NA`. 
```

**Comment by peterdesmet** [comment]
I also want to test this:

I also want to test:

```r
has_distribution(
  139334288, 
  status = "ABSENT"
  country = "CA"
)
```

Should return TRUE, even though there are distributions with status = PRESENT

**Comment by damianooldoni** [comment]
Thanks @peterdesmet! I didn't think about the fact that multiple distributions were possible. Actually it makes sense, for example when we want to specify different `status` values (where a taxa is absent and where it is present like in your example). Or different `establlishmentMeans` for different areas. I will give a look, it seems to me not a bad bug to solve! 
So, the next two calls of `has_distribution` should both return TRUE:
```r
has_distribution(
  139334288, 
  status = "ABSENT"
  country = "CA"
)
has_distribution(
  139334288, 
  status = "PRESENT"
  country = "CA"
)
```

**Comment by damianooldoni** [comment]
Functionality added in 6151503bea5294b33e852a828c99cc18254e23ad and succesfully tested in 9dfd90d5b727b39a5e697b1dc2ad3e51ff7851e3.
I implemented it as a for loop on all rows of `distr_properties`. I couldn't implement it in a more dplyr-like way (piping). Why? I found impossible to implement it and still mantaining the functionality of parsing multiple values within single property value (e.g. a distribution where `country = "BE,NL"`). Working with `rowwise()` doesn't work because any command after it gives the error that the input is not a data.frame (rowwise() is a kind of group_by: df seems the same, but actually its class changes, it is a `rowwose_df`). If we assume that multiple values never happen, then the code becomes much easier. But I would not risk.
